Zend/Optimizer/zend_inference.c

```diff
@@ -2409,6 +2409,11 @@ ZEND_API uint32_t zend_fetch_arg_info_type(const zend_script *script, const zend
 	return zend_convert_type(script, arg_info->type, pce);
 }
 
+static uint32_t zend_fetch_class_const_type(const zend_script *script, const zend_class_constant *class_const, zend_class_entry **pce)
```

Comment:
This function seems a bit redundant.

Reply:
Yes, it looks like, but there is a `zend_fetch_arg_info_type()` and a `zend_fetch_prop_type()` function with nearly the same structure, so I thought this is the consistent way to use `zend_convert_type()`

Reply:
I still think it should be dropped. We can drop the other one too in a separate PR if you wish.

Reply:
OK, I can separately get rid of the other ones.